transport/workflow-client-server-secure.md

```diff
@@ -2,148 +2,148 @@
-`2020.3 (2020.3.34 minimum and above),  2021.3 (2021.3.1f1 minimum and above) and 2022.1 (2022.1.0b16 minimum and above)`
+The Unity Transport package can be configure to encrypt the connection between the server and the client while ensuring the server's/client's authenticity.
+
+Secure connections are available with editor versions 2020.3 (starting at 2020.3.34), 2021.3, and 2022.1 and above.
 
 ## Server authentication
 
 :::warning Warning
-Be sure private keys are not included in your code base or client builds.
+This example uses hardcoded certificates to make understanding the process easier, but in a real deployment the server certificates should be kept separate from client builds. One way to achieve this is to put them on a separate assembly, or load them from a file on the server.
 :::
 
 ### High level authentication process
-In this configuration, the server will provide a certificate to the client (`certificate`) containing the server host name.
-The client will compare the server hostname to the one it knows (`serverName`) and will then validate the provided certificate against its own root certificate (`caCertificate`) confirming the server identity.
+
+In this configuration, the server will provide a certificate to the client (`certificate`) to prove its identity. The client will validate the certificate against its own root certificate (`caCertificate`) to validate its identity.
 
 :::note
-Root certificate is also sometimes referred as CA Certificate.
+Root certificates are also sometimes referred to as CA certificates.
 :::
 
-Once its identity confirmed, the server will then use the private key (`privateKey`) to establish the secure communication.
+Once its identity confirmed, the server will then use the private key (`privateKey`) to establish the secure communication channel.
 
 ### Requirements
-To use the client server secure workflow, you need a `valid certificate` and the `root certificate` it has been generated from. You also need the `private key` that has been used to create the certificate.
-If you don't have these, they can be generated using OpenSSL. The procedure is detailed hereafter. 
+
+To use the client/server secure workflow, you need a valid certificate and the root certificate that was used to sign it. You also need the private key that has been used to create the certificate. If you don't have these, they can be generated using OpenSSL. The procedure is detailed hereafter. 
 
 ### Generating the required keys and certificates with OpenSSL
 
 It is assumed that you have [OpenSSL](https://www.openssl.org/) installed on your machine.
 
-#### Generate the Certification Authority root certificate. 
-First thing first is to generate a private key. We will use it later on to generate the Certification Authority root certificate. 
+#### Generate the root certificate
+
+First thing first is to generate a root private key. We will use it later on to generate the root certificate. 
+
```

Comment:
> First, generate a root private key. You will need it later to generate the root certificate.

I changed "we" to "you" because it's poor practice to switch between perspectives. The [Unity Style guide recommends using second-person ("you")](https://docs-style-guide.unity.com/style/first-second-and-third-person). 

Although, I'm not too worried about many of these comments because the same type of corrections should be applied throughout the entire documentation set.